FABulous/FABulous_CLI/FABulous_CLI.py

```diff
@@ -851,21 +858,34 @@ def do_run_simulation(self, args):
 
         except sp.CalledProcessError:
             logger.error("Simulation failed")
-            remove_dir(tmpDir)
+            remove_dir(buildDir)
             return
 
-        make_hex(bitstreamPath, bitstreamPath.with_suffix(".hex"))
-
+        # bitstream hex file is ues for simulation so it'll be created in the test directory
+        bitstreamHexPath = (buildDir.parent / bitstreamPath.stem).with_suffix(".hex")
+        if self.verbose or self.debug:
+            logger.info(f"Make hex file {bitstreamHexPath}")
+        make_hex(bitstreamPath, bitstreamHexPath)
         vvp = check_if_application_exists(os.getenv("FAB_VVP_PATH", "vvp"))
+
+        # $plusargs is used to pass the bitstream hex and waveform path to the testbench
+        vvpArgs = [
+            f"+output_waveform={testPath / topModule}.{waveform_format}",
+            f"+bitstream_hex={bitstreamHexPath}",
+        ]
+
         try:
-            runCmd = [f"{vvp}", f"{bitstreamPath.parent}/{vvpFile}"]
+            runCmd = [f"{vvp}", f"{buildDir}/{vvpFile}"]
```

Comment:
#303 mentioned need the `-fst` to work. Changing it now or I do a PR later to fix it?

Reply:
Since it's just a tiny fix, we could actually just do it here, good point. The following worked for me:
```suggestion
            runCmd = [f"{vvp}", f"{buildDir}/{vvpFile}", "-fst"]
```

Reply:
It is more than this since it should be added based on the command input.

Reply:
Ah yeah sure, my bad.

Reply:
I've added -fst to the CLI and the makefile.